packages/core/content-releases/server/src/services/release-action.ts

```diff
@@ -243,16 +243,32 @@ const createReleaseActionService = ({ strapi }: { strapi: Core.Strapi }) => {
         return acc;
       }, []);
 
-      const contentTypeModelsMap = contentTypeUids.reduce(
-        (
-          acc: { [key: ReleaseAction['contentType']]: Struct.ContentTypeSchema },
+      const workflowsService = strapi.plugin('review-workflows').service('workflows');
+
+      const contentTypeModelsMap = await contentTypeUids.reduce(
+        async (
+          accPromise: Promise<{
+            [key: ReleaseAction['contentType']]: Struct.ContentTypeSchema & {
+              requiredStageToPublish?: number;
+            };
+          }>,
           contentTypeUid: ReleaseAction['contentType']
         ) => {
-          acc[contentTypeUid] = strapi.getModel(contentTypeUid);
+          const acc = await accPromise;
+          const contentTypeModel = strapi.getModel(contentTypeUid);
+
+          const workflow = await workflowsService.getAssignedWorkflow(contentTypeUid, {
+            populate: 'stageRequiredToPublish',
+          });
+
+          acc[contentTypeUid] = {
+            ...contentTypeModel,
+            requiredStageToPublish: workflow?.stageRequiredToPublish,
```

Comment:
couldnt be a more minor comment, but could we unify the naming of `stageRequiredToPublish` and `requiredStageToPublish` ?

Reply:
You're right, my mistake 😅